presto-product-tests/src/main/java/com/facebook/presto/tests/cassandra/DataTypesTableDefinition.java

```diff
@@ -28,10 +28,9 @@
 import java.util.List;
```

Comment:
it looks like we should merge this commit with commit before, as the previous would not pass product tests